stand-alone templates/laureate-template.html

```diff
@@ -0,0 +1,554 @@
+      <!--Fetching the image from wikidata external datasource-->
+      <semantic-query query='
+        PREFIX owl: <http://www.w3.org/2002/07/owl#>
+        PREFIX wdt: <http://www.wikidata.org/prop/direct/>
+        PREFIX wd: <http://www.wikidata.org/entity/>
+        PREFIX wiki: <https://wikidata.metaphacts.com/>
+        SELECT (REPLACE(SAMPLE(STR(?images)), "http", "https", "i") AS ?image) WHERE {
+          BIND (?? as ?subject)
+          ?subject owl:sameAs ?entity.
+          FILTER (!isBlank(?entity))
+          SERVICE wiki:sparql {
+            ?entity wdt:P18 ?images.
+          }
+        }' template='{{>tpl }}'>
+        <template id='tpl'>
+          {{#each bindings}}
+          <div>
+            <img height='365' width='auto' src='{{image.value}}' alt=''>
+          </div>
+          {{/each}}
+        </template>
+      </semantic-query>
+      <div style='padding: 0px 16px 16px 16px;'>
+        <p><span class="material-symbols-outlined" style='font-size: 64px;'>school</span></p>
+        <h1 style='font-size: xx-large;'><mp-label iri='{{page-resource}}'></mp-label></h1>
+        <!--fetching the short description about the laureate from wikidata and filtering out all, but those in English-->
+        <semantic-query
+          query='
+            PREFIX owl: <http://www.w3.org/2002/07/owl#>
+            PREFIX : <http://www.metaphacts.com/resource/>
+            PREFIX wiki: <https://wikidata.metaphacts.com/>
+            PREFIX wdt: <http://www.wikidata.org/prop/direct/>
+            PREFIX wd: <http://www.wikidata.org/entity/>
+            PREFIX schema: <http://schema.org/>
+
+            SELECT DISTINCT (SAMPLE(?desc) as ?description) WHERE {
+              ?? owl:sameAs ?obj.
+              FILTER (!isBlank(?obj))
+              SERVICE wiki:sparql {
+                ?obj schema:description ?desc.
+                FILTER LANGMATCHES( lang(?desc), "en" )  
+              }
+            }' 
+          template='{{> template}}'>
+        <template id='template'>
+          {{#each bindings}}
+          <h4 style='margin-top: -8px; color: #bb8a35;'>{{description.value}}</h4><br>
+          {{/each}}
+        </template>
+        </semantic-query>
+        <!--Fetching the motivation that is provided for each awarded Nobel prize-->
+        <semantic-query query='
+          SELECT DISTINCT ?nobel ?motivation WHERE {
+            BIND (?? as ?subject)
+            ?subject <http://data.nobelprize.org/terms/laureateAward> ?laureateAward.
+            ?laureateAward <http://data.nobelprize.org/terms/motivation> ?motivation.
+            ?laureateAward ^<http://purl.org/dc/terms/hasPart> ?nobel.
+            FILTER(LANG(?motivation)="en")
+          }' template='{{>tpl }}'>
+          <template id='tpl'>
+            {{#each bindings}}
+            <div style='display: flex; flex-direction: row;' >
+              <div style='padding-right: 8px;'>
+                <img height='35' src='/assets/nobel.svg' alt=''>

```

Comment:
Replace the `<img>` with material symbols - `<span class="material-symbols-outlined" style='font-size: 35px;'>award_star</span>`